### ISSUE TITLE
1 Unknown returncode

### ISSUE THREAD

**Issue by schumi2004** [polini/TvheadendMobileUI#12]
When installed and opening the html pages (index, mobile etc) it returns error:

1 Unknown returncode

TVH version used 3.9.1281

**Comment by detlef-bloch** [comment]
Switching from XBIAN 1Alpha5 to a fresh install of 1RC2 to run on a Rhaspberry PI B I did  a git clone for both tvheadend and TvheandendMobilUI. Compiling and installing tvheadend went without any problems: HTS Tvheadend 3.9.1338~g7fe33f7 Providing all setup for .hts from the 1A5 Backup rendered the PI into the trusted DVB-T receiver/recorder I enjoyed before.
But the  TvheandendMobilUI did not work as expected and experienced before. Errors like this one, but also missing locations (tvheadend started from commandline, not as a service) were prompted in large numbers. Inspection of its git revealed, that solely the Master branch became exported during cloning, the TvheadendMobile 4.0 branch is left out. Because of API changes in Tvheadend 3.9xxx this version 4 is needed to function properly. I was lucky to have an old version 4 fileset on the 1A5 Backup, copied it into the proper location and everything works now ok.

Hopefully this branch gets included to be clone-able to non-dev users, as trying to "download" this tree with the help of HTTrack returns regularly this message:

Error:  "Forbidden" (403) at link https://assets-cdn.github.com/ (from https://github.com/polini/TvheadendMobileUI/tree/tvheadend-4.0)
